### DISCUSSION TITLE
Ledger Live Support for ADA

### DISCUSSION THREAD

**Discussion by TrackRunny** [LedgerHQ/ledger-live-desktop#3420] [Ideas]
Any possibilities of Ledger Live Support for ADA Coin? Its really popular and number 7 on coinmarket cap, and its a shame to not see it have Ledger Live support quite yet.

**Comment by Kintaro** [comment]
Now #3 and still no support. Can we get this on the roadmap?

**Reply by CallMeCom** [reply]
This is a vicious circle!  We ask Ledger for Ledger Live support and then they say this is something IOHK, Emurgo, or any other 3rd party must deal with...... we go back to them and they say they can not control a wallet someone else creates and this is totally up to Ledger.   Both are content to sit and point the finger at one another about whos fault it is that the users are getting a sub-standard experience and when we voice it they are more than happy to dismiss us and tell us it isn't their problem......  interesting way of going at this guys!

To be honest I'm not very impressed with either project at this point and am now looking into diversifying my substantial cardano holdings into other promising projects and moving to a hardware wallet that includes more functionality in the software wallet.  I am sure this will be the end result for a lot of us.

**Reply by DarkIntegrity** [reply]
I don't know how this would be IOHK's problem, Free hot wallets like Atomic already have full ADA support, highly doubt IOHK went ahead and implemented the support for Atomic. Atomic wallet allows sending, receiving and staking Cardano. Cardano is at a 50B market cap right now, as well they are top 10 Market Cap wise.

**Reply by johnpc** [reply]
It's not true at all. Trust Wallet multicoin wallet did it and it's open source https://github.com/trustwallet/wallet-core. See discussion here https://github.com/LedgerHQ/ledger-live-desktop/discussions/4010

**Comment by bartuss7** [comment]
Any updates?
Cardano is in the top 10 cryptocurrencies for a long time and is the only one (top 10) without ledger live support.

**Comment by labor1983** [comment]
Since it got officially funded by catalyst and some talks between ledger, emurgo and iohk? have taken place … are their any actions going on?
